docs/source/en/quicktour.md

```diff
@@ -215,40 +215,46 @@ For the quicktour, you'll instantiate the [`DDPMScheduler`] with it's [`~diffuse
 ```py
 >>> from diffusers import DDPMScheduler
 
->>> scheduler = DDPMScheduler.from_config(repo_id)
+>>> scheduler = DDPMScheduler.from_pretrained(repo_id)
```

Comment:
```suggestion
>>> scheduler = DDPMScheduler.from_config(repo_id, subfolder="scheduler")
```

Reply:
It still says "/usr/local/lib/python3.10/dist-packages/diffusers/configuration_utils.py:239: FutureWarning: It is deprecated to pass a pretrained model name or path to `from_config`.If you were trying to load a scheduler, please use <class 'diffusers.schedulers.scheduling_ddpm.DDPMScheduler'>.from_pretrained(...) instead. Otherwise, please make sure to pass a configuration dictionary instead. This functionality will be removed in v1.0.0.
  deprecate("config-passed-as-path", "1.0.0", deprecation_message, standard_warn=False)".